lib/spaces/spaces/models/space.rb

```diff
@@ -61,6 +61,10 @@ def writing_name_for(model)
       "#{path_for(model)}/#{model.file_name}"
     end
 
+    def payload_path_for(model)
+      path_for(model).join('payload.*')
```

Comment:
I presume this also needs to change to `template.json` in order match the other commit.

Reply:
ok

Reply:
change committed